Gems/WhiteBox/Code/Source/EditorWhiteBoxComponentMode.cpp

```diff
@@ -478,6 +517,7 @@ namespace WhiteBox
         // create and register the buttons
         m_defaultModeButtonId = RegisterClusterButton(m_modeSelectionClusterId, "SketchMode");
         m_edgeRestoreModeButtonId = RegisterClusterButton(m_modeSelectionClusterId, "RestoreMode");
+        m_transformModeButtonId = RegisterClusterButton(m_modeSelectionClusterId, "select_object");
```

Comment:
nit: Update `select_object` to `TransformMode` potentially?

Reply:
umm assets ?

Reply:
Aaah got you, sorry is this referring to an existing asset for the icon? Hmm we can try and ask UX to get us a new one, in the meantime this is okay to stay as-is then, sorry for the confusion